src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -264,6 +266,23 @@ public static void load() {}
                 return ValueTypeInteger.ValueInteger.of(a.getRawValue() - 1);
             }).build());
 
+
+    /**
+     * Number ABS operator with one input and one output
+     */
+    public static final IOperator NUMBER_ABSOLUTE = REGISTRY.register(OperatorBuilder
```

Comment:
I would make this part of the `ARITHMETIC OPERATORS`.

Don't use `OperatorBuilder` directly, but create `OperatorBuilders.ARITHMETIC_1` (A variant of `OperatorBuilders.ARITHMETIC_2`).

Make sure to move this declaration to right after `ARITHMETIC_MINIMUM`.